wildstore-cli/src/main/java/edu/sjsu/wildstore/Main.java

```diff
@@ -123,32 +123,31 @@ public void search(@CommandLine.Parameters(paramLabel = "query") String query,
         }
 
         @CommandLine.Command
-        public void clean(@CommandLine.Parameters(paramLabel = "limit") int limit,
-                          @CommandLine.Parameters(paramLabel = "hostname") String hostname,
-                          @CommandLine.Option(names = "--token") String token,
-                          @CommandLine.Option(names = "--dryrun", negatable = true, defaultValue = "true")
-                          boolean dryrun) throws InterruptedException, ExecutionException {
+        public void clean(@CommandLine.Mixin CliOptions co,
+                          @CommandLine.Parameters(paramLabel = "limit", defaultValue = "10000") int limit,
+                          @CommandLine.Option(names = "--dryrun", negatable = true) Boolean dryrun)
+                throws InterruptedException, ExecutionException {
             int offset = 0;
             LinkedMultiValueMap<String, String> parameters = new LinkedMultiValueMap<String, String>();
             parameters.add("limit", String.valueOf(limit));
             List<String> result;
             int i = 0;
-            WebClient webClient = Client.getWebClient(hostname + "/api/metadata/filepath");
+            WebClient webClient = Client.getWebClient(co.metadataURL + "/api/metadata/filepath", co.token);
+            boolean dryrunVal = (dryrun != null) ? dryrun : true;
```

Comment:
if we keep the default value i think you can remove dryrunVal and just use dryrun